tests/acceptance/workflow_test.py

```diff
@@ -112,3 +113,71 @@ def test_walk_execution_history(workflow_client, decision_client):
     decision_task = poll_decision_and_respond(decision_client)
     terminate_workflow(workflow_client, workflow_id)
     walk_execution_history(decision_client, workflow_id, decision_task.workflow_run_id)
+
+
+def test_count_open_workflow_executions(workflow_client):
```

Comment:
The line of code this comment is tagged is for the acceptence tests. This boto client is not a mock and actually calls out to SWF. Although our current travis ci integration doesn't run them properly due to https://github.com/Yelp/pyswf/issues/19

https://github.com/Yelp/pyswf/blob/master/tests/acceptance/conftest.py#L42 shows how the boto client is created in a pytest conftest under the acceptance folder.